updateQDKVersion.ps1

```diff
@@ -0,0 +1,61 @@
+$ver = $args[0]
```

Comment:
Last suggestion for the top of the file, but it's often helpful to put a brief documentation comment at the top of the script so that it can be used together with integrated help tools such as `Get-Help`, intelliSense in VS Code and PowerShell ISE, or dotnet-interactive's Jupyter-based help commands.
```suggestion
<#
    .SYNOPSIS
        Updates the contents of this repo to use a given version of
        the Quantum Development Kit.
        
    .PARAMETER Version
        The version that this repo should be updated to. This version should be
        a valid NuGet package version as well as a valid tag for the
        mcr.microsoft.com/quantum/iqsharp-base Docker image.
        
    .EXAMPLE
    
        PS> ./Update-QDKVersion.ps1 -Version 0.10.2002.2610
#>
$ver = $args[0]
```